Add solution to all 5 tasks

Deman4ig opened the pull request.

Commit:
Add solution to all 5 tasks

jsru commented:
Добавляю преподавателя (@SinimaWath) для код-ревью.

SinimaWath commented on 1-module/4-task/index.js:
```diff
@@ -1,3 +1,11 @@
-  // ваш код...
+  const spamList = ['1xBet', 'XXX'];
+
+  for (let word of spamList) {
```
Все таки, стоит лучше обьявлять переменную word через const, ты же не собираешься ее изменять => значит это const

Deman4ig replied:
согласен! написал по принципу for (let i = ...)

SinimaWath commented on 1-module/5-task/index.js:
```diff
@@ -1,3 +1,5 @@
 function truncate(str, maxlength) {
-  // ваш код...
+  const result = (str.length <= maxlength) ? str : `${str.slice(0, maxlength - 1)}…`;
+
+  return result;
```
Такую запись можно сократить без промежуточной перемнной и сразу вернуть значение: `return ....`

Deman4ig replied:
пофиксил

SinimaWath commented on 1-module/5-task/index.js:
```diff
@@ -1,3 +1,5 @@
 function truncate(str, maxlength) {
-  // ваш код...
+  const result = (str.length <= maxlength) ? str : `${str.slice(0, maxlength - 1)}…`;
```
Тут не нужны скобки, в тернарном операторе, сначала выполниться условие: `str.length <= maxlength ? ... : ...`

Deman4ig replied:
и тут тоже пофиксил

Commit:
Update tasks #4 #5 with the suggestions:

Task #4 - replcase let with const in for clause
Task #5 - return result w/o a var; remove brackets from ternary clause;

jsru commented:
Добавляю преподавателя (@SinimaWath) для код-ревью.
